module/Entra/AdditionalFunctions/Disconnect-Entra.ps1

```diff
@@ -1,3 +1,6 @@
+# ------------------------------------------------------------------------------
+#  Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.  See License in the project root for license information.
+# ------------------------------------------------------------------------------
```

Comment:
I think we need to leave a blank line after the license information

Reply:
Updated